sources/reference/admiral-db.md

```diff
@@ -0,0 +1,99 @@
+
+```
+|___ These values are easy to set now, but hard to change later! Please confirm that they are correct:
+Installer Access Key:     AKIAI...
+Installer Secret Key:     zD5SlYS...
+Admin Panel Address:      165.227.8.141
+Database Address:         165.227.8.142
+Database Type:            New
+Database Port:            5432
+Database Password:        ay3-ay3_capt4in!
+|___ Enter Y to confirm or N to re-enter these values.
+Y
+|___ Confirmation received
+|___ Saving values
+|___ Saved access key
+|___ Saved secret key
+|___ Saved admin panel address
+|___ Saved database address
+|___ Saved database installation type
+|___ Saved database port
+|___ Saved database password
+
+```
+
+## Remote machine setup
+
+Once you confirm that your values are correct, if you elected to install a new database, Shippable will print a command that must be run on your database machine.  This will grant ssh access to the installer so that it can connect and run the installation script.  Make sure to run this command on your database machine before you move on to the next step.
+
+```
+|___ Run the following command on 165.227.8.142 to allow SSH access:
+sudo mkdir -p /root/.ssh; echo ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCzwSBD2um+6IQg5pTaxln61ByIyTQwf5EbdG+lkCVwKeWOPnGEQvMsHepBhi15eG7icpx1m6Fx+azFAxuxofmmhwCvlZPsqImRzCNCMKrUf0GxdlMjvt5JD5WbMr5WYpPtzh0z7g7Be1DbssrwF6tsmhoJc3AAGoKhdf1pgMHwAuL5i1KxBmSheCJ6SPFctHpct6XuUvVCxRX7Ty6MSqaV+rAvDyGPGBsnN5EKpQriLfMVnTVbblrUw2/ZwEqS2uj+CeHJ9yOxk+cc9/iKio4otuEHfBHwBgEJPfys1liQoPfHar7TIttSfMzB7Xe/nF root@admiral-main >> /root/.ssh/authorized_keys;
+|___ Enter Y to confirm that you have run this command
+Y
+|___ Confirmation received
+
+```
+
+Once you confirm that you've run the command, the installer will proceed to download relevant images and perform the basic setup requirements for running Shippable.
+```
+|___ Admiral container successfully running
+|___ Go to 165.227.8.141:50003 to access the admin panel
+|___ Login Token: aacfae98-0cf2-42b1-bc62-0738c3505d74
+|___ Installation successfully completed !!!
+
+```
+
+If you ssh into your database machine, you should be able to connect to it by using the `psql` utility.
+```
+root@admiral-db:~# psql shipdb -U apiuser -h localhost
+Password for user apiuser:
+psql (9.5.7)
+SSL connection (protocol: TLSv1.2, cipher: ECDHE-RSA-AES256-GCM-SHA384, bits: 256, compression: off)
+Type "help" for help.
+
+shipdb=> \dt
+
+```
+
+## Checking the logs
+Once the Installation is complete, you can view the database logs by visiting the admiral UI in your browser, and logging in with the provided token.
+
+<img src="../../images/reference/admiral/admiral-login.png" alt="login screen">
+
+This will take you to the main configuration panel, where you can see a "database" section.  On the right side, you can select the "config" button to view some of the settings, and you can select the "logs" button to see the logs that were produced during database installation.
```

Comment:
Maybe initialization instead of installation?  Since we don't have installation logs.